introduction/from_flat_php_to_symfony2.rst

```diff
@@ -370,7 +370,7 @@ on the requested URI::
         echo '<html><body><h1>Page Not Found</h1></body></html>';
     }
 
-For organization, both controllers (formerly ``index.php`` and ``show.php``)
+For organization, both controllers (formerly ``/index.php`` and ``/index.php/show``)
```

Comment:
Sorry but I don't follow you. This is what I'm trying to do:

![controller-names](https://cloud.githubusercontent.com/assets/73419/25422416/1d3d34e8-2a61-11e7-9f3d-b5a31db96f33.png)